Experimental/MultiplayerUseCases/Assets/Common/Scripts/NetworkManagerUI.cs

```diff
@@ -106,6 +120,15 @@ void OnSceneLoaded(Scene loadedScene, LoadSceneMode arg1)
 
         void SetNetworkPortAndAddress(ushort port, string address, string serverListenAddress)
         {
+            if (string.IsNullOrEmpty(address) || !NetworkEndPoint.TryParse(address, port, out NetworkEndPoint networkEndPoint))
+            {
+                Debug.LogError($"IP address '{address}' or port '{port}', are not valid. Reverting IP address to {k_DefaultIP} and reverting port to {k_DefaultPort}");
+                address = k_DefaultIP;
+                port = k_DefaultPort;
+                m_AddressInputField.SetValueWithoutNotify(k_DefaultIP);
+                m_PortInputField.SetValueWithoutNotify(k_DefaultPort.ToString());
+            }
```

Comment:
1. Having another round of validation here is good, but you should also do it at input time as users have a chance to fix it immediately. That's the same reasoning of web pages telling you if a password is not strong enough the moment you try to input it, not after you've done everything else.
2. That validation logic is not complete. What happens if I write "150.pizza.487.unity" as an ip address?